web/leaderboard/src/app/queries/ChainContext.tsx

```diff
@@ -0,0 +1,118 @@
+import { createContext, useContext, useState, useEffect } from "react";
+import {
+  useLeaderboardRanking24Hours,
+  useByUserLeaderboardRanking24Hours,
+  LeaderboardRanking,
+  LeaderboardRankingRes,
+  queryLeaderboardRankingAllTime,
+  useLeaderboardRankingAllTime,
+  useByUserLeaderboardRankingAllTime,
+} from "./useLeaderboardRanking";
+import { useQuery } from "@apollo/client";
+import { useAccount, useConnect, useDisconnect, useEnsAvatar } from "wagmi";
+
+export type Network = "arbitrum" | "etheruim" | "solana";
```

Comment:
ethereum misspelled, also this should exist somewhere else (probably in surfing)